src/test/java/net/snowflake/client/jdbc/ResultSetIT.java

```diff
@@ -955,4 +955,51 @@ public void testGetTimeNullTimestampAndTimestampNullTime() throws Throwable {
       }
     }
   }
+
```

Comment:
EmptyResultSet class is never used except for the one function getGeneratedKeys(). Since our driver does not implement generating keys, an empty ResultSet class is returned.